scripts/core/common.yml

```diff
@@ -462,6 +462,10 @@ etors:
       desc: $x_memory_sub_allocations_exp_properties_t
       version: "1.5"
       value: "0x0002000D"
+    - name: IMAGE_VIEW_PLANAR_EXT_DESC
+      desc: $x_image_view_planar_ext_desc_t
+      value: "0x0002000E"
+      version: "1.5"
```

Comment:
I don't think we want to mix standard and experimental extensions like this so I would recommend the following:

1. Move the definition of this enum up before any of the "experimental" structure types, so just after `DEVICE_IP_VERSION_EXT`.
2. Do not assign the explicit enum value `0x0002000E` and use the automatic enum value instead.